xarray/tests/test_cftime_offsets.py

```diff
@@ -1406,8 +1406,8 @@ def test_date_range_like(start, freq, cal_src, cal_tgt, use_cftime, exp0, exp_pd
         elif "YS" in freq:
             freq = freq.replace("YS", "AS")
             expected_pandas_freq = freq
-        elif "Y-" in freq:
-            freq = freq.replace("Y-", "A-")
+        elif "YE-" in freq:
+            freq = freq.replace("YE-", "A-")
```

Comment:
It's maybe a bit confusing, but at the moment the failure is only with pandas versions >= 2.2.0, for which this code is not run (it needs to be changed for other reasons in #8629).  Feel free to punt on this for now, since @max-sixty xfailed this test in #8637 (go ahead and merge main), and we'll fix it in #8629.